.azure-pipelines/alpha.yml

```diff
@@ -0,0 +1,63 @@
+trigger:
```

Comment:
What does "alpha" imply?

Reply:
"alpha" in this case means the current head of master for cpython. I'm open to other names. However, "nightly" seems too ambiguous.

Reply:
I think alpha is a little too generic as well. How about "tests-nightly-python".